app/main/views/sign_out.py

```diff
@@ -1,12 +1,37 @@
-from flask import redirect, url_for
+import os
+
+import requests
+from flask import current_app, redirect, url_for
 from flask_login import current_user
 
 from app.main import main
 
+# ask login.gov if we really need manual logout and what's up with one hour sessions
+# ask login.gov how they recommend approaching dev environment
+# ask Tim Donaworth the same for #2
+
+
+def _sign_out_at_login_dot_gov():
+    base_url = "https://idp.int.identitysandbox.gov/openid_connect/logout?"
+    client_id = f"client_id={os.getenv('LOGIN_DOT_GOV_CLIENT_ID')}"
+    post_logout_redirect_uri = "post_logout_redirect_uri=http://localhost:6012/sign-out"
```

Comment:
Would it make sense to have the `base_url` and `post_logout_redirect_uri` come from environment variables as well, like the others that were added to the `sample.env` file?  I'm a bit leery of having them hardcoded in like this, even though I know it's strictly for the development environment.